crates/ty_server/tests/e2e/main.rs

```diff
@@ -197,7 +193,7 @@ impl TestServer {
 
         let workspace_configurations = workspaces
             .into_iter()
-            .map(|(folder, options)| (folder.uri, options))
+            .filter_map(|(folder, options)| Some((folder.uri, options?)))
```

Comment:
Isn't it required that we have one option for each `url`? What's the reason for allowing `None` for the configuration?